stock-filters/ChirpVillage/Surface.py

```diff
@@ -22,6 +22,9 @@ def __init__(self, level, box):
         self.surface_map = self.init_surface_map(level)
         self.door_blocks = []
 
+    def set_surface_map(self, surface_map):
```

Comment:
Why is this needed? its public and can be modified and set directly

Reply:
meant to take this out